src/js/player.js

```diff
@@ -851,14 +856,9 @@ class Player extends Component {
     }
 
     // Initialize tech instance
-    let TechComponent = Tech.getTech(techName);
+    const TechClass = Tech.getTech(techName);
 
-    // Support old behavior of techs being registered as components.
-    // Remove once that deprecated behavior is removed.
-    if (!TechComponent) {
-      TechComponent = Component.getComponent(techName);
```

Comment:
I don't know if we show check `getComponent` at all. I think we should just throw an error if `TechComponent`/`TechClass` does not exist. We could check `getComponent` here but I think that we should be able to have a Tech and a Component with the same name (as weird as that would be)